### PR TITLE
Angular2 Tutorial

### PR DESCRIPTION
**Pull request by barbatus**
Update 4 steps of the Angular2 tutorial to be consistent the new Angular2-Meteor package.
Forth step is only half ready.

Deployed here - http://ng-meteor.meteor.com.

### CODE REVIEW

File: .docs/angular-meteor/packages/tutorial-step-diff-compiler/plugin.js [hunk 1]
```diff
@@ -22,6 +22,14 @@ function multiPatchHandler(compileStep) {
   });
 }
 
+function prepareSummary(message) {
```
Comment (Urigo) [inline .docs/angular-meteor/packages/tutorial-step-diff-compiler/plugin.js]: @barbatus are those changes should go in the https://github.com/meteor/tutorial-tools/ packages?

**Comment by MilosStanic** [comment]
I tried doing this exactly exactly as you described, except  I was unable to do `meteor add urigo:angular2-meteor` before manually cloning the `angular2-meteor` repo into my `/packages` folder. 
But I get following messages, and meteor can't go up: 
```
$ meteor
[[[[[ ~/Work/chatapp ]]]]]                    

=> Started proxy.                             
=> Started MongoDB.                           
client/app.ts (2, 42): Cannot find module 'angular2/angular2'.
                                              
events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: Cannot find module 'reflect-metadata' from '/home/milos/Work/chatapp/packages/angular2/.npm/package'
    at /home/milos/.meteor/packages/cosmos_browserify/.0.7.0.nk14f9++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:46:17
    at process (/home/milos/.meteor/packages/cosmos_browserify/.0.7.0.nk14f9++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:173:43)
    at ondir (/home/milos/.meteor/packages/cosmos_browserify/.0.7.0.nk14f9++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:188:17)
    at load (/home/milos/.meteor/packages/cosmos_browserify/.0.7.0.nk14f9++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/milos/.meteor/packages/cosmos_browserify/.0.7.0.nk14f9++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /home/milos/.meteor/packages/cosmos_browserify/.0.7.0.nk14f9++os+web.browser+web.cordova/plugin.CosmosBrowserify.os/npm/CosmosBrowserify/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:108:15)
```

Btw, I'm using node 4.1.1.

**Comment by barbatus** [comment]
@MilosStanic There is some issue going on with cosmos:browserify@0.7.0, so just reverted it back to 0.5.0 version. It should work now, please check it out.

**Comment by MilosStanic** [comment]
Seems to work now, thanks. Also would be nice to be able to install urigo:angular2-meteor via meteor add, instead of git cloning it first. Thanks a lot!

**Comment by barbatus** [comment]
@Urigo Please check http://ng-meteor.meteor.com out. I just finished updates to the first 4 parts of the tutorial. Especially polished 'Blaze-like Binding to Angular2' and part devoted to the server side.
Also re-read multiple times everything.

**Pull request merged**